lib/matplotlib/axes/_axes.py

```diff
@@ -3010,9 +3021,20 @@ def pie(self, x, explode=None, labels=None, colors=None,
             raise ValueError("Wedge sizes 'x' must be non negative values")
 
         sx = x.sum()
-        if sx > 1:
-            x = x / sx
 
+        if normalize is None:
+            if sx < 1:
+                cbook.warn_deprecated(
+                    "3.3", message="normalize=None does not normalize "
+                    "if the sum is less than 1 but this behavior"
+                    "is deprecated since %(since)s until %(removal)s. "
+                    "After the deprecation "
+                    "period the default value will be normalize=True. "
+                    "To prevent normalization pass normalize=False ")
+            else:
+                normalize = True
+        if normalize:
+            x = x / sx
```

Comment:
What would happen if `normalize = False and sx > 1`? Unlike `normalize = False and sx < 1` (which will look the same as the old behavior), we don't have any precedent for what the output should look like. To me, it doesn't make sense to make a pie plot that that more than 100%, since some of the pies will overlap each other (making it look like a normal pie plot, but where some of the slices *look* the incorrect size).

I think it would be less surprising to the user if we fail when asked to make pie plots with `normalize = False and sz > 1`.

- `normalize = False and sx < 1`, a partial pie, this is existing behavior.
- `normalize = False and sx > 1`, more than a full pie....it's not clear to me what this should look like once drawn....and it's new behavior
- `normalize = True and sx >= 1`, the old behavior is unchanged....
- `normalize = True and sz < 1`, the new behavior, I definitely like this being built-in to `pie`.

Reply:
We recently made some changes to `pie` to not accept negative values (which also resulted in a greater-than-full pie),

Reply:
TIL. I guess it's part of the pie now.